bin/crew

```diff
@@ -1877,6 +1880,10 @@ def is_command(name)
   return !!!name[/^[-<]/]
 end
 
+trap('INT') do
+  abort 'Interrupted.'.lightred
+end
```

Comment:
Why is this needed?

Reply:
This is used to hide the additional backtrace message when `ctrl+c` is pressed (since this is not an error, backtrace messages are unnecessary):
```
# without trap
$ ruby -e 'sleep 10'
^C-e:1:in `sleep': Interrupt
        from -e:1:in `<main>'

# with trap
$ ruby -e 'trap("INT") { abort "This hide the backtrace" }; sleep 10'
^CThis hide the backtrace
```

Reply:
I'm not sure this is a good idea.  There might be an interrupt internally and traceback would be helpful in those cases.

Reply:
As I know the interrupt signal is reserved as a user signal (should be fired by user only)

Reply:
Alright then I'll just pass the folks who want to troubleshoot apps to you then. :)